src/assets/icons.js

```diff
@@ -0,0 +1,3 @@
+export * from 'react-icons/ai';
```

Comment:
Suggestion: This export will increase the bundle size, I think you should just import the needed ones in the respective page
you can try doing build and compare bundle size with and without this file